migrations/versions/e417a13e136d_venue_venueroom_sequence.py

```diff
@@ -0,0 +1,64 @@
+"""added sequence to venue and venue_room
+
+Revision ID: e417a13e136d
+Revises: c3069d33419a
+Create Date: 2019-02-07 09:58:40.632722
+
+"""
+
+revision = 'e417a13e136d'
+down_revision = 'c3069d33419a'
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.sql import table, column
+
+project = table('project',
+    column('id', sa.Integer())
+    )
+
+venue = table('venue',
+    column('id', sa.Integer()),
+    column('project_id', sa.Integer()),
+    column('seq', sa.Integer()),
+    column('created_at', sa.DateTime())
+    )
+
+venue_room = table('venue_room',
+    column('id', sa.Integer()),
+    column('venue_id', sa.Integer()),
+    column('seq', sa.Integer()),
+    column('created_at', sa.DateTime())
+    )
+
+
+def upgrade():
+    op.add_column('venue', sa.Column('seq', sa.Integer(), nullable=True))
+    op.add_column('venue_room', sa.Column('seq', sa.Integer(), nullable=True))
+
+    connection = op.get_bind()
+
+    # update venue sequences
+    projects = connection.execute(sa.select([project.c.id]))
+    project_ids = [proj_tuple[0] for proj_tuple in projects]
+
+    for project_id in project_ids:
+        venues = connection.execute(sa.select([venue.c.id]).where(venue.c.project_id == project_id).order_by('created_at'))
+        venue_ids = [venue_tuple[0] for venue_tuple in venues]
```

Comment:
Not a big deal, but this can be simpler:

```python
venue_ids = [r[0] for r in connection.execute(sa.select([venue.c.id]).where(venue.c.project_id == project_id).order_by('created_at'))]
```